php/class-fieldmanager-group.php

```diff
@@ -350,6 +350,11 @@ public function presave( $values, $current_values = array() ) {
 			}
 		}
 
+		// Handle empty groups.
```

Comment:
I wonder whether it's worth adding some details to this comment, or even examples of the scenarios in which it's needed. "Empty groups" could be confused with the `group_is_empty` property, which I don't think (?) is intended.

Reply:
Good point, this is actually for handling groups that have a multiple select in them, and possible other scenarios -- I'm going to try to flush out what other scenarios that would include.